docs/content/manual/pre-release/ha/single-replica-node-down.md

```diff
@@ -24,7 +24,9 @@ https://github.com/longhorn/longhorn/issues/2309
 2. Create deployment|statefulset for volume.
 3. Power down node of volume/replica.
 4. Volume will become faulted.
-5. Wait for pod deletion & recreation on another node. For RWX volume power on the node. 
-6. Verify the auto salvage finishes for volumes.
-7. Verify volume attached & accessible by pod (i.e test data is available).
+5. Wait for 7 min for pod deletion & recreation on another node.
```

Comment:
Curious, why 7 minutes?

Reply:
After turning off the node of volume/replica, I observed that 7 minutes later the pod deletion & recreation would start. I just update this information on the test steps.

Reply:
Got it, thank you for the clarification.

7 minutes is very specific and a little misleading. Could you add a comment mentioning that the pod recreation will not happen immediately?

Reply:
This should be related to `--pod-eviction-timeout` of kube control manager. You can mention that, but no need to have a specific number here. As Chinya said, it will be a little confusing. 

Also, why removing `For RWX volume power on the node`?

Reply:
Sounds good :+1: